reference/promotions.v3.yml

```
@@ -320,6 +320,24 @@ paths:
                   example: 5
                   minimum: 0
                   maximum: 100000
+                prefix:
+                  pattern: '[A-Z0-9_-]'
+                  type: string
+                  description: 'The fixed text or characters that will appear at the beginning of every generated coupon code. Only capital letters, numbers, underscores and hyphens are allowed.'
+                  example: PRE-
```

🍹 probably keep example without "_" and "-" so we don't have to update again when adding the "separator" later ? 
```suggestion
                  example: PRE
```

---

Done